python_modules/dagster/dagster/core/definitions/decorators/op_decorator.py

```diff
@@ -181,7 +181,7 @@ def _resolve_output_defs_from_outs(
 
 
 @overload
-def op(name: Callable[..., Any]) -> SolidDefinition:
+def op(name: Callable[..., Any]) -> OpDefinition:
```

Comment:
When using a type annotation imported as a future, you need to use quotations around the annotation:
```suggestion
def op(name: Callable[..., Any]) -> "OpDefinition":
```